### PR TITLE
Catty-165: SetBackgroundBrick doesn't work

### PR DESCRIPTION
**Pull request by neel-makhecha**
After the necessary changes, SetBackgroundBrick now works as intended.

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catty/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [ ] Include the name of the Jira ticket in the PR’s title
- [ ] Verify that the Jira ticket is in the status *Ready for Development*
- [ ] Include a summary of the changes plus the relevant context
- [ ] Choose the proper base branch (*develop*)
- [ ] Confirm that the changes follow the project’s coding guidelines
- [ ] Verify that the changes generate no compiler or linter warnings
- [ ] Perform a self-review of the changes
- [ ] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Confirm that new and existing unit tests pass locally
- [ ] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [ ] Stick to the project’s git workflow (rebase and squash your commits)
- [ ] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *#catty* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Comment by m-herold** [comment]
Thank you very much! This "typo" had major consequences. So in order to avoid that in the future: could you please add a UI test (_BrickCellTests.swift_) verifying that the picker for this Brick actually shows all looks/backgrounds? Maybe you can do it similar to the control bricks (_testChangeControlDone_ and _testChangeControlCancel_)? Any other suggestion is of course also welcome.

**Comment by neel-makhecha** [comment]
> Thank you very much! This "typo" had major consequences. So in order to avoid that in the future: could you please add a UI test (_BrickCellTests.swift_) verifying that the picker for this Brick actually shows all looks/backgrounds? Maybe you can do it similar to the control bricks (_testChangeControlDone_ and _testChangeControlCancel_)? Any other suggestion is of course also welcome.

Hi Michael, 

I have added a new test that'll first add new looks and then it will check whether they all are present in the picker when the brick is tapped. Please review it.

File: src/Catty UITests/ScriptCollectionView/BrickCellTests.swift [hunk 1]
```diff
@@ -25,6 +25,8 @@ import XCTest
 class BrickCellTests: XCTestCase {
 
     var app: XCUIApplication!
+    let testLookNames_toType: [String] = ["_test1", "_test2", "_test3"]
```
Comment (m-herold) [inline src/Catty UITests/ScriptCollectionView/BrickCellTests.swift:28]: Your test looks very nice, thanks! What is the difference between `testLookNames_toType` and `testLookNames`? I can not figure it out when reading the variable names. Do we need both?

File: src/Catty/PocketPaint/ViewController/PaintViewController.m [hunk 2]
```diff
@@ -183,6 +183,8 @@ - (void)setupCanvas
     NSInteger height = (NSInteger)self.projectHeight;
     CGRect rect = CGRectMake(0, 0, width, height);
     self.drawView = [[UIImageView alloc] initWithFrame:rect];
+    self.drawView.accessibilityIdentifier = @"Canvas";
```
Comment (m-herold) [inline src/Catty/PocketPaint/ViewController/PaintViewController.m:186]: Why do we need this here? Is this ticket related to Paint?

File: src/Catty UITests/ScriptCollectionView/BrickCellTests.swift (hunk 1 above)
Comment (neel-makhecha) [reply src/Catty UITests/ScriptCollectionView/BrickCellTests.swift:28]: Yes, we will need both. 
When a new look is created and saved, an alert appears with a text field to save it with a name. By default that text field already have a text "look". So previously, I had used backspace character and removed it and then used values from _testLookNames_. But it made the code look more complicated for no reason (Because, Swift don't have \b that's why we will need another variable which can store unicode equivalent for it). So then I decided to append __test#_  from _testLookNames_toType_ at the end and then when adjusting picker, we will need _testLookNames_.

File: src/Catty/PocketPaint/ViewController/PaintViewController.m (hunk 2 above)
Comment (neel-makhecha) [reply src/Catty/PocketPaint/ViewController/PaintViewController.m:186]: For creating a new look, I used 'Draw new image' option. When doing that, I was required to make changes on the drawView in order to save it. So I had to tap drawView. So this is to identify drawView from that test.

File: src/Catty UITests/ScriptCollectionView/BrickCellTests.swift (hunk 1 above)
Comment (m-herold) [reply src/Catty UITests/ScriptCollectionView/BrickCellTests.swift:28]: I see! Would maybe `alert3.textFields.buttons["Clear text"].tap()` be a potential solution for this? One minor suggestion: could you please reorder your private methods to the top (below `tearDown`) so that all actual test functions are next to each other?

File: src/Catty/PocketPaint/ViewController/PaintViewController.m (hunk 2 above)
Comment (m-herold) [reply src/Catty/PocketPaint/ViewController/PaintViewController.m:186]: That's cool! Could you here please also include the "Paint" keyword so that this identifier will be more readable from "outside"? (e.g. `PaintCanvas` or similar).

File: src/Catty UITests/ScriptCollectionView/BrickCellTests.swift (hunk 1 above)
Comment (neel-makhecha) [reply src/Catty UITests/ScriptCollectionView/BrickCellTests.swift:28]: Thank you so much! :-) 
I completely missed the existence of that clear button.

**Comment by neel-makhecha** [comment]
Hi Michael, committed changes as per your suggestions. Please review it.

**Comment by m-herold** [comment]
Thanks!
